js/ui/map.js

```diff
@@ -57,47 +57,51 @@ var defaultOptions = {
 };
 
 /**
- * Creates a map instance. This is usually the beginning of your map:
- * you tell Mapbox GL JS where to put the map by specifying a `container`
- * option, and the map's style with `style` and other attributes of the map,
- * and in return Mapbox GL JS initializes the map on your page and returns
- * a map variable that lets you programmatically call methods on the map.
+ * The `Map` object represents the map on your page. It exposes methods
+ * and properties that enable you to programmatically change the map,
+ * and fires events as users interact with it.
+ *
+ * You create a `Map` by specifying a `container` and other options.
+ * Then Mapbox GL JS initializes the map on the page and returns your `Map`
+ * object.
+ *
  * @class Map
  * @param {Object} options
- * @param {string|Element} options.container HTML element to initialize the map in (or element id as string)
- * @param {number} [options.minZoom=0] Minimum zoom of the map
- * @param {number} [options.maxZoom=20] Maximum zoom of the map
- * @param {Object|string} [options.style] Map style. This must be an an object conforming to the schema described in
- * the [style reference](https://mapbox.com/mapbox-gl-style-spec/), or a URL to a JSON style. To load a style from the
+ * @param {Element|string} options.container The HTML element to initialize the map in, or the element's string `id`.
```

Comment:
BTW totally fine if you want to go to lowercase, was just explaining the rationale.